.github/CONTRIBUTING.md

```diff
@@ -5,4 +5,30 @@ feel it's ready to merge. Even if there's no real disagreement about a PR, at
 least one other person on the team needs to look over a PR before merging. The
 purpose of this review requirement is to ensure shared knowledge of the app and
 its changes and to take advantage of the benefits of working together changes
-without any single person being a bottleneck to making progress.
+without any single person being a bottleneck to making progress.  
+
+## Steps  
+
+1. Fork this repository.  
+2. Clone the repository by running this command:  
+    ```shell
+    $ git clone https://github.com/YOUR-GITHUB-USERNAME/playtime.git
+    ```  
+3. Add a remote upstream in order for git to know the original Playtime repository is:  
```

Comment:
This sentence has some grammatical errors. Maybe something like "Add a remote `upstream` that points to the original Playtime repository:"?